### ISSUE TITLE
[Question] Would clusters of proteins from MMSEQS2 be considered orthologs?

### ISSUE THREAD

**Issue by jolespin** [soedinglab/MMseqs2#584]
Let's say you had 100 strains of a particular species. If you did an all-vs-all clustering of proteins, would those resulting clusters be considered orthologs?

**Comment by martin-steinegger** [comment]
No, we do not filter for orthology in our clustering. They are homologous.

**Comment by martin-steinegger** [comment]
Closing this one due to inactivity — it was answered above and we haven't heard back. If it's still relevant, please reopen with your MMseqs2 version, the exact command, and the full output, and we'll be happy to help. Thanks!
